src/main/kotlin/net/atos/zac/smartdocuments/rest/RESTMappedSmartDocumentsTemplate.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.smartdocuments.rest
+
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import java.util.UUID
+
+@NoArgConstructor
+@AllOpen
+data class RESTMappedSmartDocumentsTemplate(
```

Comment:
```suggestion
data class RestMappedSmartDocumentsTemplate(
```

To comply with the Kotlin coding conventions. :-)

Reply:
Fixed